### PR TITLE
[docstring] Fix docstring for AltCLIPTextConfig, AltCLIPVisionConfig and AltCLIPConfig

### LINKED ISSUE #27127: I will take AltCLIPVisionConfig, AltCLIPTextConfig
I will take AltCLIPVisionConfig, AltCLIPTextConfig

_Originally posted by @AksharGoyal in https://github.com/huggingface/transformers/issues/26638#issuecomment-1769186608_

### PR DESCRIPTION
**Pull request by AksharGoyal**
# What does this PR do?

 - Fixed docstrings for AltCLIPTextConfig, AltCLIPVisionConfig and AltCLIPConfig
 - Cleaned few docstrings

Fixes #26638

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

File: utils/check_docstrings.py [hunk 1]
```diff
@@ -79,7 +79,7 @@
     "AlbertTokenizerFast",
     "AlignTextModel",
     "AlignVisionConfig",
-    "AltCLIPTextConfig",
+    # "AltCLIPTextConfig",
```
Comment (amyeroberts) [inline utils/check_docstrings.py:82]: This should just be removed rather than commented out 

```suggestion
```

**Review by amyeroberts — APPROVED** [review]
Thanks for adding all these fixes! 

Just a small comment. Once resolved we're good to merge 🤗

File: utils/check_docstrings.py (hunk 1 above)
Comment (AksharGoyal) [reply utils/check_docstrings.py:82]: Ok I will remove that. I guess I can do the same for VisionConfig too as they were in the same file.

**Comment by AksharGoyal** [comment]
Hi @amyeroberts I have made the change. Let me know if anything else is needed from my side.

**Comment by amyeroberts** [comment]
@AksharGoyal Thanks! For the removal of the `AltCLIPVisionConfig` what errors are you getting?

**Comment by AksharGoyal** [comment]
@amyeroberts This is what I see when I go to one of the [failed workflows](https://app.circleci.com/pipelines/github/huggingface/transformers/76713/workflows/18e1d257-983d-4f62-8733-8827d768d425/jobs/975389)
```
2023-10-29 19:05:35.437483: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
/home/circleci/transformers/src/transformers/deepspeed.py:23: FutureWarning: transformers.deepspeed module is deprecated and will be removed in a future version. Please import deepspeed modules directly from transformers.integrations
  warnings.warn(
Traceback (most recent call last):
  File "utils/check_docstrings.py", line 1231, in <module>
    check_docstrings(overwrite=args.fix_and_overwrite)
  File "utils/check_docstrings.py", line 1223, in check_docstrings
    raise ValueError(error_message)
ValueError: There was at least one problem when checking docstrings of public objects.
The following objects docstrings contain templates you need to fix: search for `<fill_type>` or `<fill_docstring>`.
- AltCLIPVisionConfig

Exited with code exit status 1

```

**Comment by amyeroberts** [comment]
@AksharGoyal The issue is arising because some of the parameters in `AltCLIPVisionConfig`'s  docstring aren't correctly filled in. You'll see now in the configuration file `<fill_docstring>` has been added in places where descriptions are needed

**Review by amyeroberts — APPROVED** [review]
Thanks for iterating!

**Pull request merged**